docpages/install/install-arch-aur.md

```diff
@@ -15,6 +15,8 @@ or use your favourite package manager:
 yay -Sy dpp
```

Comment:
always upgrade when you refresh your package db to avoid partial update that can brick your OS. Use `-Syu` instead of `-Sy`

Reply:
i resolved it but it was -Sy by default